src/OrbitCaptureClient/CaptureEventProcessor.cpp

```diff
@@ -82,6 +82,10 @@ void CaptureEventProcessor::ProcessEvent(const ClientCaptureEvent& event) {
     case ClientCaptureEvent::kSystemMemoryUsage:
       // TODO (http://b/179000848): Process the system memory usage information.
       break;
+    case ClientCaptureEvent::kApiEvent: {
+      api_event_processor_.ProcessApiEvent(event.api_event());
```

Comment:
Could you add one (or more) tests in CaptureEventProcessorTest?

Reply:
Please see new ApiEventProcessorTest.cpp

Reply:
I think it would still be beneficial to have a small test in `CaptureEventProcessorTest.cpp` that covers the fact that `kApiEvent`s are handled properly. Then of course all the details of `ApiEventProcessor` are tested in isolation in `ApiEventProcessorTest.cpp`.